packages/editor-ui/src/components/CredentialCard.vue

```diff
@@ -20,151 +149,20 @@
 			</n8n-text>
 		</div>
 		<template #append>
-			<ProjectCardBadge :resource="data" :personal-project="projectsStore.personalProject" />
-			<div ref="cardActions" :class="$style.cardActions">
-				<n8n-action-toggle :actions="actions" theme="dark" @action="onAction" @click.stop />
+			<div :class="$style.cardActions" @click.stop>
+				<ProjectCardBadge :resource="data" :personal-project="projectsStore.personalProject" />
+				<n8n-action-toggle
+					data-test-id="credential-card-actions"
+					:actions="actions"
+					theme="dark"
+					@action="onAction"
+					@click.stop
```

Comment:
Is this still needed if there's a `@click.stop` on the wrapper div?

Reply:
no, removed